static/css/restyle.css

```diff
@@ -0,0 +1,695 @@
+/* Hide outlines Firefox shows on every click */
+* {
+  outline: none;
```

Comment:
That's not ideal for accessibility especially people that navigate by keyboard unless you use alternative styling where relevant. See http://a11yproject.com/posts/never-remove-css-outlines/ for more info and suggestions.